code/_onclick/hud/plane_master.dm

```diff
@@ -15,6 +19,7 @@
 //Why do plane masters need a backdrop sometimes? Read http://www.byond.com/forum/?post=2141928
 //Trust me, you need one. Period. If you don't think you do, you're doing something extremely wrong.
 /obj/screen/plane_master/proc/backdrop(mob/mymob)
+	addtimer(CALLBACK(src, TYPE_PROC_REF(/atom, add_filter), "displacer", 1, displacement_map_filter(render_source = GRAVITY_PULSE_RENDER_TARGET, size = 10)), 2 SECONDS)//Why a timer vs just apply on initialize / async? I don't know. It just can't be, neither works correctly. Don't lower below 2 seconds unless you can see effects through walls with no issue.
```

Comment:
I think this is suppose to be "displace" and not "displacer"? https://www.byond.com/docs/ref/#/{notes}/filters/displace

Also why not put this on `/obj/screen/plane_master/gravpulse/backdrop` instead?

Reply:
It needs to apply to all planes, otherwise it only effects that plane, and nothing else. Putting it on the floor plane for example, would make it show only on floors. Walls / mobs / items would be unaffected.

Reply:
Alright that makes sense.